docs/examples/01-green-hydrogen.ipynb

```diff
@@ -77,14 +77,14 @@
       "cell_type": "markdown",
       "metadata": {},
       "source": [
-        "#### GreenHEART YAML Overview\n",
+        "#### H2Integrate YAML Overview\n",
         "\n",
-        "The GreenHEART YAML file includes information required for simulating the renewable energy generation through the end product production (hydrogen, steel, ammonia). Such as:\n",
+        "The H2Integrate YAML file includes information required for simulating the renewable energy generation through the end product production (hydrogen, steel, ammonia). Such as:\n",
         "\n",
         "- **Plant Design Scenarios**: Specifies location and connection of individual components of plant.\n",
         "- **Financial Parameters**: General financial parameters needed for calculating the levelized cost of product and other financial metrics.\n",
         "- **Hydrogen Equipment Information**: Including electrolyzer specifications, compressor, pipeline, and H<sub>2</sub> storage information.\n",
-        "- **Policy Options**: Describes options to include [IRA](https://www.whitehouse.gov/wp-content/uploads/2022/12/Inflation-Reduction-Act-Guidebook.pdf) tax incentives policy in GreenHEART simulation."
+        "- **Policy Options**: Describes options to include [IRA](https://www.whitehouse.gov/wp-content/uploads/2022/12/Inflation-Reduction-Act-Guidebook.pdf) tax incentives policy in H2Integrate simulation."
```

Comment:
```suggestion
        "- **Policy Options**: Describes options to include [IRA](https://bidenwhitehouse.archives.gov/wp-content/uploads/2022/12/Inflation-Reduction-Act-Guidebook.pdf) tax incentives policy in H2Integrate simulation."
```

Reply:
Updated in the examples root folder as well.